16/umbraco-cms/customizing/extending-overview/extension-types/menu-item.md

```diff
@@ -0,0 +1,263 @@
+---
+description: >-
+    Create menu items that appear throughout the backoffice, in sidebars, button flyouts, and more.
+---
+
+# Menu Items
+
+Menu Item extensions are used in combination with [Menu](menu.md) extensions. Menu items can be placed in custom menus, sidebars, and even added into the built-in/default Umbraco menu extensions. Extension authors can use the default Menu Item component or create their own custom Menu Items and register them as extensions.
+
+<figure><img src="../../../.gitbook/assets/menu-item.png" alt="" width="250"><figcaption><p>Menu Item</p></figcaption></figure>
+
+Menu Item extensions can be defined using either JSON in `umbraco-package.json` or using TypeScript.
+
+## Creating Menu Items
+
+Menu Item extensions can be created using either JSON or TypeScript. Both approaches are shown below.
```

Comment:
This line is almost the same as the one above. Can we remove one of them?

"Menu Item extensions can be defined using either JSON in `umbraco-package.json` or using TypeScript."
"Menu Item extensions can be created using either JSON or TypeScript. Both approaches are shown below."